buildpack/bp_descriptor_test.go

```diff
@@ -118,11 +118,11 @@ func testBpDescriptor(t *testing.T, when spec.G, it spec.S) {
 			h.AssertEq(t, descriptor.Stacks[0].ID, "some.non-magic.value")
 			h.AssertEq(t, len(descriptor.Targets), 1)
 			h.AssertEq(t, len(descriptor.Targets), 1)
-			h.AssertEq(t, descriptor.Targets[0].Arch, "amd64")
+			h.AssertEq(t, descriptor.Targets[0].Arch, "*")
 			h.AssertEq(t, descriptor.Targets[0].OS, "linux")
 			h.AssertEq(t, len(descriptor.Targets[0].Distributions), 0)
 		})
-		it("detects both windows/amd64 if batch files are present and linux", func() {
+		it("detects both windows/* if batch files are present and linux", func() {
```

Comment:
sorry, this might be a bad test-case name.  the test case has files with no extension (`build`) for linux as well as files with .bat extension (`build.bat`) for windows.  does that help?

Reply:
we figured it out!